test/test_regression_cli.py

```diff
@@ -36,6 +46,18 @@ def getmd5(filename):
         sum.update(line)
     return sum.hexdigest()
 
+def getmd5d(directoryename):
```

Comment:
Let's expand this function name a little (on the first read my brain kept thinking this was the same as `getmd5`). Even `getmd5dir` would be fine.

Also, here and below it should be `directoryname` instead of `directoryename`.